docs/source/release/v6.7.0/diffraction.rst

```diff
@@ -32,45 +32,44 @@ Engineering Diffraction
 New features
 ############
 - The GSASII tab outputs the sample logs attached to the relevant focused data, as specified in the Engineering Diffraction settings. This behaves in the same way as before for the fitting tab.
-- Check peak-window signal-to-noise ratio before attempting to fit a diffraction peak in :ref:`FitPeaks <algm-FitPeaks>`
+- Add check for ``peak-window``:``signal-to-noise`` ratio before attempting to fit a diffraction peak in :ref:`FitPeaks <algm-FitPeaks>`.
```

Comment:
This came from #35094 and needs to be in the Powder section. In addition I think it needs to be clarified. Something along the lines of 
```
New parameter `MinimumSignalToNoiseRatio` has been added to `FitPeaks` and `PDCalibration` algorithms - this will exclude peaks from automated fitting below a threshold signal-to-noise ratio estimated from the data in the fitting window.`
```